bin/restarter

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+MINIMUM_TIME_ALIVE = Integer(ENV.fetch("RESTART_MINIMUM_TIME_ALIVE", "3600"), 10)
+require "json"
+
+command = ARGV.dup.freeze
+start_time = Time.now
+
+iso_start = start_time.utc.strftime("%Y-%m-%dT%H:%M:%S%:z").freeze
+puts JSON.generate(restarter: {
+  start_time: iso_start,
+  startup: {
+    subprocess_args: command,
+    minimum_time_alive: MINIMUM_TIME_ALIVE
+
+  }
+})
+
+loop do
+  pid = Process.spawn(*command, out: :out, err: :err)
+  Process.wait(pid)
```

Comment:
```suggestion
  system(*command)
```

I think this has the same behavior.

Reply:
buffers & captures stdout, so it doesn't, right?

Reply:
Oh no that's backticks.